### DISCUSSION TITLE
收集你的「打呼聲資料」，調整枕頭高度！科技如何 KO 現代人睡眠障礙？

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#401] [科技新知]
「韓國 Zerema 枕頭結合 AI 深度學習來紀錄使用者的睡眠習慣，該枕頭配備 AI 驅動的打呼檢測技術，當枕頭感覺到使用者打呼時，就會自己提高或降低枕頭的高度，調整適當的呼吸形式，提供個性化的睡眠分析。」

看了報導才了解到原來睡姿會影響是否打呼。透過AI枕頭調整睡姿減緩打呼實在挺創新又有醫療意義，只是實際上市跟普及可能還需許多的驗證跟民眾的買單。

資料來源：https://buzzorange.com/techorange/2022/03/14/zerema-sleep/

**Comment by YuHengChen54** [comment]
這個感覺很讚欸，可能也可以幫助很多會呼吸中止的人~~

**Comment by lx3211** [comment]
AI在醫療方面真的有極高的發展價值欸

**Comment by NcuJerryKao** [comment]
AI還能以光纖的sensor 監測人們的脈搏心跳 一出事 第一時間能立即派發人員搶救

**Comment by BingRong1217** [comment]
AI真的好厲害，連這種事都能替我們解決

**Comment by limitx0** [comment]
好想試試那個枕頭啊

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by awkward-willy** [comment]
打呼畢竟是很多人都有的問題，這樣的設計能減少很多枕邊人或宿舍室友的困擾，很令人耳目一新。不過也很好奇若將感測器或抬升裝置等塞進枕頭中，是否會造成使用時有異物感，進而反而導致睡眠品質的下降。

**Comment by larry40305** [comment]
但我覺得調整高度所能改善的效果有限，如果真的是睡眠中止症患者應該還是要盡快就醫。

**Comment by boson13579** [comment]
Zerema會幫助人們更有效的睡眠，是個頗具實用性的發明。
